projects/addon-doc/src/components/main/main.component.ts

```diff
@@ -25,8 +25,9 @@ import {Subject} from 'rxjs';
 })
 export class TuiDocMainComponent {
     night =
-        this.windowRef.matchMedia('(prefers-color-scheme: dark)').matches ||
-        this.storage.getItem('night') === 'true';
+        this.storage.getItem('night') === 'true' ||
+        (this.storage.getItem('night') === null &&
+            this.windowRef.matchMedia('(prefers-color-scheme: dark)').matches);
```

Comment:
This fixes the overriding of chosen theme if the dark mode is preferred by the system.
Previously the dark mode turned on every time I reloaded the page.